commit 66db639f2c6c1b9dda8cf18e2a212f160b268a3b
Author: Tomas Korbar <tkorbar@redhat.com>
Date:   Thu Jul 23 09:35:56 2020 +0200

    Search for libyang.h in build directory

diff --git a/Doxyfile.in b/Doxyfile.in
index 2534da3..272248e 100644
--- a/Doxyfile.in
+++ b/Doxyfile.in
@@ -781,7 +781,7 @@ WARN_LOGFILE           =
 # spaces. See also FILE_PATTERNS and EXTENSION_MAPPING
 # Note: If this tag is empty the current directory is searched.
 
-INPUT                  = @CMAKE_BINARY_DIR@/src/libyang.h \
+INPUT                  = ./build/src/libyang.h \
                          ./src/tree_data.h \
                          ./src/tree_schema.h \
                          ./src/extensions.h \
